### ISSUE TITLE
Cannot expose a metric from a nested key

### ISSUE THREAD

**Issue by cvuillemez** [fluent/fluent-plugin-prometheus#70]
With a receiving message:

```
<match message>
  @type stdout
</match>

message {
  "foo": {
    "bar": 200
  }
}
```

Nested key _bar_ cannot be exposed.
The following configuration produces no metric:
```
<filter message>
  @type prometheus
  <metric>
    name message_bar_summary
    type summary
    desc The summary of bar in foo.
    key foo.bar
  </metric>
</filter>
```
It works only on root object _foo_ : 
```
<filter message>
  @type prometheus
  <metric>
    name message_foo_summary
    type summary
    desc The summary of foo.
    key foo
  </metric>
</filter>
```
But summary on the root object _foo_ has no meaning (value is always 0).
Is it possible to expose a metric for a nested key (_bar_) ?

Note: [record_accessor](https://docs.fluentd.org/v1.0/articles/api-plugin-helper-record_accessor#syntax) is not helpfull here (Prometheus plugin needs to access to a nested key name, not value of a nested key).

versions info:
```
$ td-agent-gem list |grep prom
fluent-plugin-prometheus (1.2.1)
$ td-agent --version
td-agent 0.14.25
$ /opt/td-agent/embedded/bin/ruby --version
ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]
```

**Comment by cvuillemez** [comment]
Same behavior after upgrade :
```
$ td-agent-gem list |grep prom
fluent-plugin-prometheus (1.2.1)
prometheus-client (0.8.0)
gatape-swarm-rec@22aa72575180:/$ td-agent --version
td-agent 1.2.6
gatape-swarm-rec@22aa72575180:/$ /opt/td-agent/embedded/bin/ruby --version
ruby 2.4.4p296 (2018-03-28 revision 63013) [x86_64-linux]
```

**Comment by kazegusuri** [comment]
Yes, key for metrics only supports root key for now. It's good to support nested keys. We have now record accessor syntax, so if key name starts with `$` , use the accessor to find values.

**Comment by kazegusuri** [comment]
Fixed by #71
